homeassistant/components/vicare/strings.json

```diff
@@ -83,6 +83,8 @@
             "state": {
               "permanent": "permanent",
               "ventilation": "schedule",
+              "standby": "[%key:common::state::standby%]",
+              "standard": "grundbetrieb",
               "sensor_driven": "sensor",
               "sensor_override": "schedule with sensor-override"
```

Comment:
Why aren't these capitalized, and what is `grundbetrieb`. It sounds german

Reply:
I left `Grundbetrieb` in here as I need a good translation here. 
It's the minimal possible level for always-on systems.